### PR TITLE
Back to `feed_hot`/`feed_recent` + some reorg.

### PR DESCRIPTION
**Pull request by mzhurovich**
@dkorolev PTAL

### CODE REVIEW

File: server.h [hunk 1]
```diff
@@ -244,8 +241,11 @@ class CTFOServer {
             return card_entry;
           };
 
+          std::vector<std::reference_wrapper<std::vector<ResponseCardEntry>>> feeds;
```
Comment (dkorolev) [inline server.h:244]: :+1: cute!

File: server.h [hunk 2]
```diff
@@ -345,7 +310,7 @@ class CTFOServer {
     } else {
       if (entry.m != "TICK") {
         DebugPrint(Printf(
-            "[OnMidichloriansEvent] Suspicious event with method '%s' (t = %ull)", entry.m.c_str(), entry.t));
+            "[OnMidichloriansEvent] Suspicious event with method '%s' (t = %llu)", entry.m.c_str(), entry.t));
```
Comment (dkorolev) [inline server.h:313]: Yep.

**Comment by dkorolev** [comment]
LGTM!

**Comment by mzhurovich** [comment]
@dkorolev Thanks!

**Pull request merged**